python/examples/integration/README.md

```diff
@@ -0,0 +1,8 @@
+An example showing how to connect gRPC's OpenTracing spans to other OpenTracing
+spans.
+
+## Usage
+```
+python integration_server.py --access_token `lightstep-access-token` &
```

Comment:
would be better to support Jaeger's (OSS) library here... it can run in a single local docker image.